packages/playground-examples/copy/ja/3-7/Fixits/Infer From Usage Changes.ts

```diff
@@ -0,0 +1,32 @@
+//// { compiler: {  noImplicitAny: false }, order: 2 }
+
+// 3.7 で TypeScript 既存の「使用状況からの推論」によるコード修正はよりスマートになりました。
+// 既知の重要な型 (string, number, array, Promise) のリストを用いて、
+// 型の使用方法がこれらのオブジェクトのAPIと一致するかどうかを推論する様になりました。
+
+// 次のいくつかの例では、関数のパラメータを選択し、電球をクリックして "Infer Parameter types..." を選択してください。
+
+// 数値配列の推論:
+
+function pushNumber(arr) {
+  arr.push(12)
+}
+
+// promiseの推論:
+
+function awaitPromise(promise) {
+  promise.then(value => console.log(value))
+}
+
+// 関数とその戻り値の型の推論:
+
+function inferAny(app) {
+  const result = app.use('hi')
+  return result
+}
+
+// 文字列配列に文字列が追加された為、文字列配列を推論します:
```

Comment:
IMO: 「に」だと思ってますが、もしかして「を」の方が一般的です？
てにをは難しい
```suggestion
// 文字列配列に文字列が追加された為、文字列配列に推論します:
```